### ISSUE TITLE
bug: VPC Flow Logs false positive, and link broken

### ISSUE THREAD

**Issue by brsolomon-deloitte** [aquasecurity/tfsec#2024]
**Describe the bug**

A two-for-one bug!

1. tfsec 1.28.1 reports `Result #6 MEDIUM VPC Flow Logs is not enabled for VPC` despite me having `enable_flow_log = true`.
2. It also displays a broken link on output and I can't seem to find this check anywhere at all in the tfsec docs.

**To Reproduce**

```
module "vpc" {
  source  = "terraform-aws-modules/vpc/aws"
  version = "3.19.0"
  ...


  # Enable VPC flow logs (default false)
  # https://docs.aws.amazon.com/vpc/latest/userguide/flow-logs.html#flow-logs-basics
  enable_flow_log                                 = true
  flow_log_destination_type                       = "cloud-watch-logs"
  flow_log_file_format                            = "plain-text"
  create_flow_log_cloudwatch_log_group            = true
  create_flow_log_cloudwatch_iam_role             = true
  flow_log_cloudwatch_log_group_retention_in_days = 7
}
```

```
$ tfsec .

...
Result #6 MEDIUM VPC Flow Logs is not enabled for VPC
─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
  terraform-aws-modules/vpc/aws/<redacted>.terraform/modules/vpc/main.tf:20-43
   via vpc.tf:4-42 (module.vpc)
─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
   20  ┌ resource "aws_vpc" "this" {
   21  │   count = local.create_vpc ? 1 : 0
   22  │
   23  │   cidr_block          = var.use_ipam_pool ? null : var.cidr
   24  │   ipv4_ipam_pool_id   = var.ipv4_ipam_pool_id
   25  │   ipv4_netmask_length = var.ipv4_netmask_length
   26  │
   27  │   assign_generated_ipv6_cidr_block = var.enable_ipv6 && !var.use_ipam_pool ? true : null
   28  └   ipv6_cidr_block                  = var.ipv6_cidr
   ..
─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
          ID aws-ec2-require-vpc-flow-logs-for-all-vpcs
      Impact Without VPC flow logs, you risk not having enough information about network traffic flow to investigate incidents or identify security issues.
  Resolution Enable flow logs for VPC

  More Information
  - https://aquasecurity.github.io/tfsec/v1.28.1/checks/aws/ec2/require-vpc-flow-logs-for-all-vpcs/
```

The link `https://aquasecurity.github.io/tfsec/v1.28.1/checks/aws/ec2/require-vpc-flow-logs-for-all-vpcs/` 404s and there seems to be no mention of a flow logs-related rule for AWS under either ec2 or vpc.

**System Info**

`tfsec` 1.28.1, Terraform v1.4.2 on MacOS 13.2.1 arm64.

**Comment by Amaladavis** [comment]
facing the same issue

**Comment by taskooh** [comment]
I also have same issue.

**Comment by PLeS207** [comment]
facing the same issue

**Comment by FireballDWF** [comment]
I've encountered the same issue

**Comment by nikpivkin** [comment]
Hi @brsolomon-deloitte !
I opened issue in Trivy https://github.com/aquasecurity/trivy/issues/4963

**Comment by frauniki** [comment]
This problem has not been resolved.
It continues in tfsec v1.28.4.

Because the `defsec` module is not updated in tfsec.
This problem has been resolved in defsec v0.92.0.
https://github.com/aquasecurity/defsec/pull/1422

Ref: https://github.com/aquasecurity/tfsec/pull/2071

**Comment by simar7** [comment]
> This problem has not been resolved. It continues in tfsec v1.28.4.
> 
> Because the `defsec` module is not updated in tfsec. This problem has been resolved in defsec v0.92.0. [aquasecurity/defsec#1422](https://github.com/aquasecurity/defsec/pull/1422)
> 
> Ref: #2071

We've moved our efforts to Trivy. Please see https://github.com/aquasecurity/tfsec/discussions/1994
